lib/id3c/cli/io/adapters.py

```diff
@@ -0,0 +1,50 @@
+"""
+Utilities for interfacing with various systems.
+"""
+from googleapiclient import discovery
+from google.oauth2 import service_account
+from typing import Tuple
+from os import environ
+from hashlib import sha1
+import pandas as pd
+import json
+
+GOOGLE_SERVICE_ACCOUNT_CREDS_ENV_VAR = "GOOGLE_SERVICE_ACCOUNT_CREDENTIALS_FILE_PATH"
```

Comment:
Storing this string as a new variable feels a bit redundant .. perhaps we could just read 'GOOGLE_SERVICE_ACCOUNT_CREDENTIALS_FILE_PATH' from the environment?

Reply:
This was a super valid point. Now that Thomas showed me how the Google API can work by setting one env var we don't need these all over the place, so that is gone from the code.